torch/nn/parallel/distributed.py

```diff
@@ -305,14 +305,19 @@ class DistributedDataParallel(Module):
 
     Args:
         module (Module): module to be parallelized
-        device_ids (list of int or torch.device): CUDA devices. This should
-                   only be provided when the input module resides on a single
-                   CUDA device. For single-device modules, the i'th
-                   :attr:`module` replica is placed on ``device_ids[i]``. For
-                   multi-device modules and CPU modules, ``device_ids`` must be
-                   ``None`` or an empty list, and input data for the forward
-                   pass must be placed on the correct device. (default: all
-                   visible devices for single-device modules)
+        device_ids (list of int or torch.device): CUDA devices.
+                   1) For single-device modules, ``device_ids`` should only
+                   contain exactly one device id, which represents the only
+                   CUDA device where the input module resides.
+                   Alternatively, if ``None`` is provided, the current device
+                   will be added to ``device_ids``. This requires calling
+                   ``torch.cuda.set_device`` to set the appropriate device
+                   before calling DDP constructor.
+                   2) For multi-device modules and CPU modules,
+                   ``device_ids`` must be ``None`` or an empty list,
+                   and input data for the forward pass must be placed on the
+                   correct device.
```

Comment:
This statement is true for both points? So maybe just call it out after point 2) in a separate paragraph?

Reply:
Thx for the suggestion! Updated.